.nuspec/Microsoft.Maui.Controls.MultiTargeting.targets

```diff
@@ -45,7 +45,7 @@
     <Compile Remove="**\Windows\**\*.xaml.cs" />
     <None Include="**\Windows\**\*.xaml.cs" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
   </ItemGroup>
-  <ItemGroup Condition=" $(TargetFramework.StartsWith('uap10.0')) ">
```

Comment:
I guess these were all commented out files ? i'm not sure why this one was never negated, maybe it was meant to be but missed and unnoticed because it didn't have any impact anyway?